CONTRIBUTING.md

```diff
@@ -1,5 +1,27 @@
+# Contributing to Eclipse Open VSX Publish Extensions
+
 Thanks for stopping by and willing to contribute to `publish-extensions`! Below are some general rules to abide by when contributing to the repository.
 
+## Code of Conduct
+
+This project is governed by the [Eclipse Community Code of Conduct](CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code.
```

Comment:
Do we have a Code of Conduct we'd like to link to or insert to the repo? This is linking to a file which does not exist

Reply:
@filiptronicek Oops, I forgot to add that. I had intended to copy the one we have in [open-vsx.org](https://github.com/eclipse/openvsx/blob/master/CODE_OF_CONDUCT.md). Are you able to modify this PR or do you need me to?